akka-docs/src/test/java/jdocs/config/ConfigDocTest.java

```diff
@@ -67,4 +74,15 @@ public ActorSystem createConfiguredSystem() {
     // #custom-config-2
     return system;
   }
+
+  public Boolean compileOnlyIsLicenseKeyValid() {
+    //#check-is-key-valid
+    ActorSystem<Void> actorSystem = ActorSystem.create(rootBehavior, "name");
+    Optional<LocalDate> licenseKey = actorSystem.getLicenseKeyExpiry();
+    if(licenseKey.isEmpty()) return false;
+
+    ChronoLocalDate today = LocalDate.from(ZonedDateTime.now());
+    return licenseKey.get().isBefore(today);
```

Comment:
shouldn't this be written as if it was a junit test, since that is what we describe in docs?
```
assertTrue(licenseKey.get().isBefore(today));
```